### ISSUE TITLE
Changes to list under "Why is this important?"

### ISSUE THREAD

**Issue by dboudreau** [w3c/wai-tutorials#368]
I get why we also throw in something about search engine in there, but we shouldn't lump that into the same bulleted list that describes some of the benefits for people. Collateral benefits of accessibility, such as those enjoyed in terms of search engine indexation have merit, but should be presented as a separate thing.

Consequently, I would remove the last bullet from that list, replace it with one that simply says "Etc." (as there could be many other examples), and add a paragraph under the list, stating the following:

"There are additional benefits to accessible page structure, beyond those experienced by people with disabilities. As an example, search engines can use the data to make better predictions on the real content of a web page."

**Comment by yatil** [comment]
I like this change and will add it to the tutorial!

**Comment by yatil** [comment]
Added to the in-development version: https://w3c.github.io/wai-tutorials/page-structure/
